server/src/bin/generate_kafka_crd.rs

```diff
@@ -0,0 +1,7 @@
+use stackable_kafka_crd::KafkaCluster;
+use std::error::Error;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    println!("{}", serde_yaml::to_string(&KafkaCluster::crd())?);
+    Ok(())
+}
```

Comment:
Can you check with crd/src/main.rs. There the crd is generated and written (not to the deploy folder).
If you want the console output i would suggest to merge these two.

Reply:
The crd/src/main will also be called manually, right?
Either way: It's not ideal but I'd suggest we merge as is for now until we can finally get a generic way to do so in kube-rs.

Reply:
Yes called manually but writes into the files. So one step less ;)